ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/af-commons/v1/fieldTemplates/label.html

```diff
@@ -1,4 +1,4 @@
-<template data-sly-template.label="${@ componentId, labelValue, labelVisible, bemBlock}">
+<template data-sly-template.label="${@ componentId, labelValue, labelVisible, labelRichText, bemBlock}">
     <label for="${componentId}" class="${bemBlock}__label"
-           data-sly-test="${labelValue && (labelVisible==null || labelVisible==true)}">${labelValue}</label>
+           data-sly-test="${labelValue && (labelVisible==null || labelVisible==true)}" isTitleRichText="${labelRichText}">${labelValue @ context = labelRichText ? 'html' : 'text'}</label>
```

Comment:
why do you need isTitleRichText ? if you need it to access in JS, it should have a data attribute. HTML elements should either have BEM for CSS or data attributes for JS interactions

Reply:
agreed, changed to data-richtext attribute for interaction with js